cherrypy/process/servers.py

```diff
@@ -185,7 +185,7 @@ def _get_base(self):
         if not self.httpserver:
             return ''
         host, port = self.bind_addr
-        if getattr(self.httpserver, 'ssl_adapter', None):
+        if getattr(self,'ssl_certificate', None):
```

Comment:
Yeah, I know what the check does, but I won't accept such patch. Also, please explain me **why** this issue happens and **prove** with tests, that patch doesn't break anything. This is what I want basically,